model/Metadata/Listener/ClassDeletionListener.php

```diff
@@ -42,14 +42,14 @@ public function listen($event): void
         }
 
         try {
-            $this->getElasticSearch()->remove($event->getClass()->getUri());
+            $this->getSearchProxy()->remove($event->getClass()->getUri());
         } catch (Exception $exception) {
             $this->getLogger()->error($exception->getMessage());
         }
     }
 
-    private function getElasticSearch(): ElasticSearch
+    private function getSearchProxy(): SearchProxy
```

Comment:
Just a note to update tao-core version on composer before merge

- [ ] Update tao-core